app/views/host/_form.html.haml

```diff
@@ -1,107 +1,14 @@
-                                            "checkchange" => ""}
-            .form-group{"ng-class" => "{'has-error': angularForm.ipmi_address.$error.requiredDependsOn}"}
-              %label.col-md-2.control-label{"for" => "ipmi_address"}
-                = _("IPMI IP Address")
-              .col-md-8
-                %input.form-control#ipmi_address{"type"                => "text",
-                                                "id"                  => "ipmi_address",
-                                                "name"                => "ipmi_address",
-                                                "ng-model"            => "hostModel.ipmi_address",
-                                                "required-depends-on" => "hostModel.ipmi_userid",
-                                                "required-if-exists"  => "ipmi_userid",
-                                                "maxlength"           => 15,
-                                                "checkchange"         => ""}
-                %span.help-block{"ng-show" => "angularForm.ipmi_address.$error.requiredDependsOn"}
-                  = _("Required")
-            .form-group
-              %label.col-md-2.control-label
-                = _("MAC Address")
-              .col-md-8
-                %input#mac_address.form-control{"type"        => "text",
-                                                "name"        => "mac_address",
-                                                "ng-model"    => "hostModel.mac_address",
-                                                "maxlength"   => "#{ViewHelper::MAX_NAME_LEN}",
-                                                "checkchange" => ""}
-      %hr
-      = render(:partial => "/layouts/angular/multi_auth_credentials",
-              :locals  => {:record => @host, :ng_model => "hostModel"})
-      = render :partial => "layouts/angular/x_edit_buttons_angular"
-
-  - unless session[:host_items].nil?
-    %h3
-      = n_("Host", "Hosts", session[:host_items].length)
-      = _('Selected')
-    = _('Click on a Host to fetch its settings')
-    %table.admittable{:height => '75'}
-      %tbody
-        %tr
-          %td
-            - if session[:host_items]
-              - @embedded = true
-              = render :partial => 'layouts/gtl'
-
-:javascript
-  ManageIQ.angular.app.value('hostFormId', '#{(@host.id || (session[:host_items] && session[:host_items].join(",")))}');
-  miq_bootstrap('#form_div');
+= render :partial => "layouts/flash_msg"
+- unless session[:host_items].nil?
+  %h3
+    = n_("Host", "Hosts", session[:host_items].length)
+    = _('Selected')
+  = _('Click on a Host to fetch its settings')
+  %table.admittable{:height => '75'}
+    %tbody
+      %tr
+        %td
+          - if session[:host_items]
+            - @embedded = true
+            = render :partial => 'layouts/gtl'
+= react('HostEditForm', { :ids => (@host.id ? [@host.id] : session[:host_items]) })
```

Comment:
can avoid spaces before and after { }
`= react('HostEditForm', {:ids => (@host.id ? [@host.id] : session[:host_items])})`

Reply:
Applied the change